package.json

```diff
@@ -78,5 +79,8 @@
   "bugs": {
     "url": "https://github.com/UKHomeOffice/hods-poc/issues"
   },
-  "homepage": "https://github.com/UKHomeOffice/hods-poc#readme"
+  "homepage": "https://github.com/UKHomeOffice/hods-poc#readme",
+  "dependencies": {
+    "@not-govuk/anchor": "^0.2.9"
+  }
```

Comment:
Something's gone wrong here. - You shouldn't need to edit this file.

Reply:
Yes, this should be done at the local (component) level

Reply:
Seems to be the same as another comment raised above - Don't know how this happened but was likely due to trying to `npm install` a component from elsewhere. Have amended this now